SYCL/ESIMD/api/functional/ctors/ctor_vector.hpp

```diff
@@ -0,0 +1,145 @@
+//===-- ctor_vector.hpp - Functions for tests on simd vector constructor
+//      definition. -------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd vector constructor.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common.hpp"
+
+namespace esimd = sycl::ext::intel::experimental::esimd;
+
+namespace esimd_test::api::functional {
```

Comment:
```suggestion
namespace esimd_test::api::functional::ctors {
```
Isn't it?

Reply:
yes, applied in this commit: 252c1c76dfd200e2c2be15d1f926ec17225cf973